internal/services/network/private_endpoint_resource.go

```diff
@@ -873,6 +874,11 @@ func flattenPrivateLinkEndpointServiceConnection(serviceConnections *[]network.P
 						privateConnectionId = serverId.ID()
 					}
 				}
+				if strings.Contains(strings.ToLower(privateConnectionId), "microsoft.redis") {
```

Comment:
@tombuildsstuff Sorry about my dumb question, but we're facing the same issue this PR is supposed to fix but the `private_connection_resource_id` contains `Microsoft.Cache/Redis` / `Microsoft.Cache/redis` and not just `Microsoft.Redis` or `Microsoft.redis`. Is this a different id? Does this PR actually fix the issue then?